projects/ngx-d3timeline/src/lib/ngx-d3timeline.component.ts

```diff
@@ -103,10 +102,15 @@ export class NgxD3timelineComponent implements OnInit, AfterViewInit {
 
   @ViewChild('svgEl') svgEl: ElementRef<SVGElement>;
 
-  constructor(public timeline: NgxD3TimelineService) {}
+  constructor(
+    public timeline: NgxD3TimelineService,
+    private hostElement: ElementRef,
```

Comment:
Can this get injected into `NgxD3TimelineService`?

Reply:
yep, didn't think it would work but it does